code/notificationbuilder/src/test/java/com/adobe/marketing/mobile/notificationbuilder/internal/templates/MockAEPPushTemplateDataProvider.kt

```diff
@@ -0,0 +1,121 @@
+            .thenReturn(MOCKED_BODY)
+        Mockito.`when`(mockBundle.getString(PushTemplateConstants.PushPayloadKeys.VERSION))
+            .thenReturn(MOCKED_PAYLOAD_VERSION)
+        return mockBundle
+    }
+
+    fun getMockedAEPDataMapWithAllKeys(): MutableMap<String, String> {
+        return mutableMapOf(
+            PushTemplateConstants.PushPayloadKeys.TAG to MOCKED_TAG,
+            PushTemplateConstants.PushPayloadKeys.TEMPLATE_TYPE to PushTemplateType.BASIC.value,
+            PushTemplateConstants.PushPayloadKeys.ACTION_URI to MOCKED_ACTION_URI,
+            PushTemplateConstants.PushPayloadKeys.ACTION_TYPE to "NONE",
+            PushTemplateConstants.PushPayloadKeys.ACTION_BUTTONS to MOCKED_ACTION_BUTTON_DATA,
+            PushTemplateConstants.PushPayloadKeys.BADGE_COUNT to "5",
+            PushTemplateConstants.PushPayloadKeys.BODY to MOCKED_BASIC_TEMPLATE_BODY,
+            PushTemplateConstants.PushPayloadKeys.CHANNEL_ID to "2024",
+            PushTemplateConstants.PushPayloadKeys.EXPANDED_BODY_TEXT to MOCKED_BASIC_TEMPLATE_BODY_EXPANDED,
+            PushTemplateConstants.PushPayloadKeys.BODY_TEXT_COLOR to "FFD966",
+            PushTemplateConstants.PushPayloadKeys.IMAGE_URL to MOCKED_IMAGE_URI,
+            PushTemplateConstants.PushPayloadKeys.LARGE_ICON to MOCKED_LARGE_ICON,
+            PushTemplateConstants.PushPayloadKeys.BACKGROUND_COLOR to "FFD966",
+            PushTemplateConstants.PushPayloadKeys.PRIORITY to NotificationPriority.getNotificationPriority(MOCKED_PRIORITY),
+            PushTemplateConstants.PushPayloadKeys.VISIBILITY to MOCKED_VISIBILITY,
+            PushTemplateConstants.PushPayloadKeys.REMIND_LATER_TEXT to "remind me",
+            PushTemplateConstants.PushPayloadKeys.REMIND_LATER_TIMESTAMP to "1234567890",
+            PushTemplateConstants.PushPayloadKeys.SOUND to "bell",
+            PushTemplateConstants.PushPayloadKeys.SMALL_ICON to MOCKED_SMALL_ICON,
+            PushTemplateConstants.PushPayloadKeys.TITLE_TEXT_COLOR to "FFD966",
+            PushTemplateConstants.PushPayloadKeys.TICKER to MOCKED_TICKER,
+            PushTemplateConstants.PushPayloadKeys.VERSION to MOCKED_PAYLOAD_VERSION,
+            PushTemplateConstants.PushPayloadKeys.STICKY to "true"
+        )
+    }
+
+    fun getMockedAEPBundleWithAllKeys(): Bundle {
+        val mockBundle = mock<Bundle>()
+        Mockito.`when`(mockBundle.getString(PushTemplateConstants.PushPayloadKeys.TAG))
+            .thenReturn(MOCKED_TAG)
+        Mockito.`when`(mockBundle.getString(PushTemplateConstants.PushPayloadKeys.TEMPLATE_TYPE))
+            .thenReturn(PushTemplateType.BASIC.value)
+        Mockito.`when`(mockBundle.getString(PushTemplateConstants.PushPayloadKeys.ACTION_URI))
+            .thenReturn(MOCKED_ACTION_URI)
+        Mockito.`when`(mockBundle.getString(PushTemplateConstants.PushPayloadKeys.ACTION_TYPE))
+            .thenReturn("NONE")
```

Comment:
nit: We can use `PushTemplateConstants.ActionType.NONE.name` here

Reply:
DOne